lib/order.rb

```diff
@@ -0,0 +1,38 @@
+class Order
+
+  attr_accessor :current_order, :total
+
+  def initialize(menu)
+    @current_order = {}
+    @total = 0
+    @dishes = menu.dishes
+    @sum = 0
+  end
+
+  def add_to_order(dish, quantity)
+    raise 'Dish not available: not part of the menu' unless @dishes.include?(dish)
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)